_people/joerg_winkler.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Jörg Winkler
+category: Team
+position: Research Assistant
```

Comment:
```suggestion
position: PhD Student
```

Mitra wrote "PhD Student", is that fine with you?

Reply:
I prefer Research Assistant as this is the actual job title ("Wissenschaftlicher Mitarbeiter"), and I mention the PhD anyway in my first sentence. I think we are all more than just PhD students, no?

Reply:
Oki doki!